root/user/UserCollections.js

```diff
@@ -0,0 +1,199 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import some from 'lodash/some';
+import * as React from 'react';
+
+import Table from '../components/Table';
+import UserAccountLayout from '../components/UserAccountLayout';
+import {withCatalystContext} from '../context';
+import {unwrapNl} from '../static/scripts/common/i18n';
+import {formatPluralEntityTypeName}
+  from '../static/scripts/common/utility/formatEntityTypeName';
+import {
+  defineNameColumn,
+  typeColumn,
+  subscriptionColumn,
+  defineActionsColumn,
+} from '../utility/tableColumns';
+
+type Props = {|
+  +$c: {user: EditorT} & CatalystContextT,
+  +collaborativeCollections: CollectionListT,
+  +ownCollections: CollectionListT,
+  +user: EditorT,
+|};
+
+type CollectionListT = {|
+  +[string]: $ReadOnlyArray<CollectionT>,
+|};
+
+const collectionsListTitle = {
```

Comment:
I'd keep object types plural
```suggestion
const collectionsListTitles = {
```

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1246/commits/6ee5d3d424b1dda8aecefe6be1b9f7d2ac4092e1#diff-3f513fa6fa1fed40110c8c20d602c01fR36